src/content/structured/components/top-nav/accessibility.mdx

```diff
@@ -26,7 +26,7 @@ import topNavFig11 from "./images/fig-11-tab-order.png";
 
 ## Easy to use for everyone
 
-On top navigation layouts which contain a search button (rather than an expanded search bar), when the search button is activated the search bar will appear below, extending the height of the top navigation and shifting other content down.
+On top navigation layouts that contain a search button (rather than an expanded search bar), when the search button is activated the search bar will appear below, extending the height of the top navigation and shifting other content down.
```

Comment:
```suggestion
On top navigation layouts that contain a search button (rather than an expanded search bar), when the search button is activated the search bar will appear below. This will extend the height of the top navigation,  shifting other content down.
```